src/components/show-task-modal/show-task-modal.component.coffee

```diff
@@ -42,6 +42,7 @@ angular.module('mnoUiElements').component('mnoShowTaskModal', {
       )
 
     ctrl.send = (markAsDone)->
+      ctrl.loading = true
```

Comment:
This loader should be set to false in a `finally`

Reply:
Even though the popup is closing after ?

Reply:
The popup won't be closed if there is a server error